neps/nep-0171.md

```diff
@@ -218,21 +218,21 @@ function nft_on_transfer(
 
 ### Events
 
-NEAR and third-party applications need to track `mint`, `transfer`, `burn`, and `contract_metadata_update` events for all NFT-driven apps consistently.
+NEAR and third-party applications need to track `mint`, `update`, `transfer`, `burn`, and `contract_metadata_update` events for all NFT-driven apps consistently.
```

Comment:
A token is updated if the `owner_id` is changed, because `owner_id` is one of the Token fields: https://nomicon.io/Standards/Tokens/NonFungibleToken/Metadata. 
So, for the transfer, we have to emit 2 logs, which is weird.

Do we want to use `token_metadata_update`?

Reply:
This is a very good point. What about `nft_metadata_update` to keep the naming convention for NFT-related events?

Reply:
Just updated to `nft_metadata_update`